sdk/src/main/java/com/hcaptcha/sdk/HCaptchaHtml.java

```diff
@@ -1,3 +1,6 @@
+// Auto-generated. Don't edit it directly.
+// Modify sdk/src/main/html/hcaptcha.html instead and run `./gradlew build` instead
+
 package com.hcaptcha.sdk;
 
 import lombok.NonNull;
```

Comment:
Having the auto-generated code versioned in the git repo can introduce scenarios were behavior is not obvious.

Tests would be necessary for the generation process: e.g. this HTML is correctly created, contains some obvious html tags, such as script loading the js api and the html container where hCaptcha is rendered.

Reply:
I pushed commit https://github.com/hCaptcha/hcaptcha-android-sdk/pull/79/commits/ce06841c31d3f368471bf7b90d3e00561047e4df to ignore this auto generated file.

Please add tests outlined above @CAMOBAP

Reply:
@DSergiu I think the correct implementation (if we wanna ignore autogenerated code) should be like this https://github.com/hCaptcha/hcaptcha-android-sdk/pull/80 

> e.g. this HTML is correctly created
We have such verification with `jshint` https://github.com/hCaptcha/hcaptcha-android-sdk/blob/main/.github/workflows/ci.yml#L30

> ... script loading the js API ...

`xmllint --html --xpath "/html/body/script[2]" sdk/build/hcaptcha-form.html`

> ... the html container where hCaptcha is rendered
`xmllint --html --xpath "/html/body/div[@id = 'hcaptcha-container']" sdk/build/hcaptcha-form.html`

@DSergiu are those `xmllint` tests will be enough? Should we add anything else?

Reply:
Should be enugh. Closing this PR in favor of the other one.